api/net/Areas/Editor/Controllers/ContentController.cs

```diff
@@ -211,9 +211,7 @@ public async Task<IActionResult> AddAsync(ContentModel model)
         var user = _userService.FindByUsername(username) ?? throw new NotAuthorizedException($"User [{username}] does not exist");
         var newContent = (Content)model;
         newContent.OwnerId = user.Id;
-        if (!newContent.PostedOn.HasValue &&
-            (newContent.Status == ContentStatus.Publish ||
-            newContent.Status == ContentStatus.Published))
+        if (!newContent.PostedOn.HasValue)
```

Comment:
We now set the value when saved no matter what.  This fixes the bug where the report option to only include newly posted content would block unpublished content filters.